### PR TITLE
Upload j meter plugins

### PR DESCRIPTION
**Pull request by dpatel7**

### CODE REVIEW

File: Docker/ICAP.jmx [hunk 1]
```diff
@@ -72,10 +74,10 @@
               <connectTime>true</connectTime>
             </value>
           </objProp>
-          <stringProp name="filename">/Test/Results/ICAP.csv</stringProp>
+          <stringProp name="filename">/home/ubuntu/icap/jmeter results/icap.csv</stringProp>
```
Comment (jpoolGW) [inline Docker/ICAP.jmx:77]: File path has been changed

File: Docker/bkp_ICAP.jmx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              <connectTime>true</connectTime>
+            </value>
+          </objProp>
+          <stringProp name="filename">/Test/Results/ICAP.csv</stringProp>
+        </ResultCollector>
+        <hashTree/>
+      </hashTree>
+    </hashTree>
+  </hashTree>
+</jmeterTestPlan>
```
Comment (jpoolGW) [inline Docker/bkp_ICAP.jmx:81]: Each jmx script should be within it's own folder

File: Docker/Dockerfile [hunk 3]
```diff
@@ -36,13 +45,26 @@ ENV JMETER_HOME=/usr/share/apache-jmeter \
...
     mv apache-jmeter-${JMETER_VERSION} ${JMETER_HOME}
 
+# Install Plugin Manager
+RUN cd /tmp/ &&\
+    wget -O jmeter-plugins-manager-1.4.jar http://search.maven.org/remotecontent?filepath=kg/apc/jmeter-plugins-manager/1.4/jmeter-plugins-manager-1.4.jar && \
+    mv /tmp/jmeter-plugins-manager-1.4.jar ${JMETER_HOME}/lib/ext/jmeter-plugins-manager-1.4.jar && \
+    wget -O cmdrunner-2.2.jar http://search.maven.org/remotecontent?filepath=kg/apc/cmdrunner/2.2/cmdrunner-2.2.jar && \
+    mv /tmp/cmdrunner-2.2.jar ${JMETER_HOME}/lib/cmdrunner-2.2.jar &&\
+    wget -O json-lib-2.4-jdk15.jar https://sourceforge.net/projects/json-lib/files/json-lib/json-lib-2.4/json-lib-2.4-jdk15.jar/download && \
+    mv /tmp/json-lib-2.4-jdk15.jar ${JMETER_HOME}/lib/json-lib-2.4-jdk15.jar &&\
```
Comment (jpoolGW) [inline Docker/Dockerfile:62]: Please group wget commands and mv commands to make this more readable

**Review by jpoolGW — APPROVED** [review]
Please raise issues on board to fix the items I've commented

File: Docker/ICAP.jmx (hunk 1 above)
Comment (dpatel7) [reply Docker/ICAP.jmx]: corrected

**Pull request merged**